dotnet/MomentoExamples/MomentoApplication/Program.cs

```diff
@@ -15,7 +15,14 @@ class Program
         static void Main(string[] args)
         {
             using SimpleCacheClient client = new SimpleCacheClient(MOMENTO_AUTH_TOKEN, DEFAULT_TTL_SECONDS);
-            client.CreateCache(CACHE_NAME);
+            try
+            {
+                client.CreateCache(CACHE_NAME);
+            }
+            catch (AlreadyExistsException e)
```

Comment:
LGTM. But can you see if .NET supports catching exception withoput using the variable?

```
try {
client.CreateCache(CACHE_NAME);
} catch (AlreadyExistsException) {
// print
}
```

Reply:
It does. I'll remove the variable.